dev/tasks/README.md

```diff
@@ -80,12 +80,12 @@ submission. The tasks are defined in `tasks.yml`
 6. Install the python dependencies for the script:
 
```

Comment:
One thing that is worth nothing in this `README.md` is that you need at least one commit in your queue repo or else this happens:

```sh
Traceback (most recent call last):
  File "crossbow.py", line 413, in <module>
    crossbow(auto_envvar_prefix='CROSSBOW')
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "crossbow.py", line 356, in submit
    queue.put(job)
  File "crossbow.py", line 186, in put
    branch = self._create_branch(task.branch, files=task.files())
  File "crossbow.py", line 158, in _create_branch
    parents = parents or [self.head]
  File "crossbow.py", line 96, in head
    return self.repo.head.target
_pygit2.GitError: reference 'refs/heads/master' not found
```

Is there any way that we can automate this away? If not, make sure to add a note about it in the README.

Reply:
We don't need parent commits, it just makes the graph nicer.

Reply:
Resolved